server/grpc_service.go

```diff
@@ -59,6 +59,29 @@ var (
 	ErrNotStarted = status.Errorf(codes.Unavailable, "server not started")
 )
 
+// Header returns header metadata.
+type Header interface {
+	GetHeader() *pdpb.RequestHeader
```

Comment:
We don't need this interface. How about using `ResponseHeader` directly?

Reply:
fix